amical/_cli/api.py

```diff
@@ -74,13 +74,13 @@ def _select_association_file(args):
         date = hdr.get("DATE-OBS", None)
         ins = hdr.get("INSTRUME", None)
 
-        if target not in (None or ""):
+        if (target is not None) & (target != "Unknown"):
```

Comment:
`&`  isn't short for `and`, it's a different operator
```suggestion
        if (target is not None) and (target != "Unknown"):
```